tests/bgp/test_4-byte_asn_community.py

```diff
@@ -73,7 +74,7 @@ def setup(tbinfo, nbrhosts, duthosts, enum_frontend_dut_hostname, enum_rand_one_
     neigh_ip_bgp_sum = nbrhosts[neigh]["host"].shell('show ip bgp summary')['stdout']
     with open(bgp_id_textfsm) as template:
         fsm = textfsm.TextFSM(template)
-        dut_bgp_id = fsm.ParseText(dut_ip_bgp_sum)[0][0]
+        dut_bgp_id = fsm.ParseText(dut_ip_bgp_sum)[asic_index][0]
```

Comment:
if it's single asic, please continue with original 0, else can use asic_index, single asic default asic is None